atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/AtlasDbTestCase.java

```diff
@@ -111,8 +113,11 @@ public void setUp() throws Exception {
     }
 
     protected KeyValueService getBaseKeyValueService() {
-        return NamespacedKeyValueServices.wrapWithStaticNamespaceMappingKvs(
-                new InMemoryKeyValueService(false, PTExecutors.newSingleThreadExecutor(PTExecutors.newNamedThreadFactory(true))));
+        return TracingKeyValueService.create(
```

Comment:
This set of inlined variables is getting kind of silly - consider extracting a variable or two, particularly `Tracers.wrap(...)`, which looks like it'd fit on one line.